test/specs/test.js

```diff
@@ -1,8 +1,19 @@
-const MainPage = require('../pageobjects/mainPage');
+import mainPage from '../pageobjects/mainPage.js';
 
 describe('AccuWeather Test Suite', () => {
-    before(async () => {
-        await MainPage.open();
-        await MainPage.consentDataUsage();
+    it('consenting to data usage and allow cookie',async () => {
+        await mainPage.acceptConsentButton();
     });
 });
+
+describe('accuWeather Search Suite', () => {
+    it('click on the search box and write "New York"', async () => {
+        await mainPage.searchCity();
```

Comment:
![image](https://github.com/nhridi/Accuweather/assets/115004083/fffce8f2-b762-404b-891a-13124fe5de5a)
Where is asserts?